src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -339,9 +339,9 @@ public class Preferences {
         "lastCombatEnvironments",
         "lastCopyableMonster",
         "lastCouncilVisit",
-        "lastFriarElbowNC",
-        "lastFriarHeartNC",
-        "lastFriarNeckNC",
+        "lastFriarsElbowNC",
+        "lastFriarsHeartNC",
+        "lastFriarsNeckNC",
```

Comment:
If I am searching the code correctly the preferences as defined in default don't have the s.  That says to me that either some script is using the wrong preference or there is a bug in KoLmafia where the wrong preference is being used.